app/org/sagebionetworks/bridge/services/email/ParticipantRosterProvider.java

```diff
@@ -65,29 +63,11 @@ String createInlineParticipantRoster() {
         if (participants.size() == 0) {
             sb.append("There are no users enrolled in this study.");
         } else if (participants.size() == 1) {
-            sb.append("There is 1 user enrolled in this study:");
+            sb.append("There is 1 user enrolled in this study. Please see the attached TSV file.");
         } else {
-            sb.append("There are "+participants.size()+" users enrolled in this study:");
+            sb.append("There are "+participants.size()+" users enrolled in this study. Please see the attached TSV file.");
         }
         sb.append(NEWLINE);
-        for (int i=0; i < participants.size(); i++) {
```

Comment:
I see the for loop is being removed.  Does it mean we are not sending detailed information about individual participants?